plugin/vimux.vim

```diff
@@ -60,7 +62,7 @@ function! VimuxRunCommand(command, ...)
     let l:autoreturn = a:1
   endif
 
-  let resetSequence = s:VimuxOption('g:VimuxResetSequence', 'q C-u')
+  let resetSequence = 'g:VimuxResetSequence'
```

Comment:
Why is this (and many other similar examples) being single quoted? Shouldn't this just be a bare variable reference not a quoted string?

Reply:
You are absolutely right. Not sure how I didn't notice this. Excited but sleepy, was I.